spec/flows/under_eighteen_flow_spec.rb

```diff
@@ -21,7 +22,17 @@
     start_assessment
     fill_in_client_age_screen(choice: "Under 18")
     fill_in_level_of_help_screen(choice: "Civil controlled work or family mediation")
+    fill_in_under_18_controlled_legal_rep_screen(choice: "No")
     fill_in_aggregated_means_screen(choice: "Yes")
     confirm_screen("immigration_or_asylum")
   end
+
+  it "exits to Check your answers if it is Controlled Legal Representation work" do
```

Comment:
Could you add a "form" spec as well (/spec/forms)? We've got them for every screen, and it gives us a place to check stuff like the correct error message showing if you try to leave the form blank.

Reply:
added